aldryn_cms/urls.py

```diff
@@ -3,8 +3,12 @@
 
 from cms.sitemaps import CMSSitemap
 
+from .views import check_uninstall_ok
+
 
 urlpatterns = [
+    # Not sure if we should hardcode admin path like this
+    url(r'^admin/~cmscloud-api/check-uninstall/$', check_uninstall_ok),
```

Comment:
Let's keep it then... if it's just this occurrence it's ok, otherwise we'll have to make something more configurable...